darglint2/config.py

```diff
@@ -26,16 +26,17 @@
 
 
 def get_logger():  # type: () -> Logger
-    """Get the default logger for darglint.
+    """Get the default logger for darglint2.
 
     Returns:
-        The default logger for darglint.
+        The default logger for darglint2.
 
     """
-    return logging.getLogger('darglint')
+    return logging.getLogger("darglint2")
 
 
 POSSIBLE_CONFIG_FILENAMES = (
+    ".darglint2",
```

Comment:
@9dogs what do you think about renaming the configuration file to `.darglint2` and using `.darglint` as a convenience fallback for those migrating from `darglint` to `darglint2`?

Reply:
Sure. Although since we're going to introduce at least one breaking change (particularly, in the command name), should we bother keeping a fallback?

Reply:
One use case I can think of is when someone finds out about `darglint2` and wants to quickly test it out on a repository which already has configuration in `.darglint`.